.github/workflows/go-postsubmit.yml

```diff
@@ -0,0 +1,38 @@
+name: GoPostSubmit
+
+on:
+  push:
+    branches:
+      - main
+      - release-*
```

Comment:
Probably not an issue, but I'd like to note that having `release-*` here would require this file to be updated if we do indeed start having `release-*` branches.